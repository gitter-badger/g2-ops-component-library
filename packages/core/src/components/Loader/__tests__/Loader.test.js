import React from 'react'
import renderer from 'react-test-renderer'

import {Loader} from '../'

describe('<Loader />', () => {
  test('should render properly', () => {
    const tree = renderer.create(<Loader />)
    expect(tree).toMatchSnapshot()
  })
})
