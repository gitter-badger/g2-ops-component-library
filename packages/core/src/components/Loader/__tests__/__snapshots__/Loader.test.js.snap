// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loader /> should render properly 1`] = `
<div
  className="ms-Spinner spinner root-34"
>
  <div
    className="ms-Spinner-circle ms-Spinner--medium circle-35"
  />
</div>
`;
